dist/source/repo-creator/index.ts

```diff
@@ -5,6 +5,7 @@ export class RepoCreator {
 
 	routes = [
 		{ route: ['', 'choose'], moduleId: './choose-repository', nav: false, title: 'Choose a Template' },
+		{ route: ['', 'sponsorship'], moduleId: './sponsorship', nav: false, title: 'Sponsorship' },
```

Comment:
This route shouldn't override the default route (`''`), it should look something like:
```
{ route: 'sponsorship', moduleId: './sponsorship', nav: false, title: 'Sponsorship' },
```